### DISCUSSION TITLE
Feature request: Switch to turn off in-app web pages

### DISCUSSION THREAD

**Discussion by Solomon1732** [amugofjava/anytime_podcast_player#75] [Ideas]
Currently, when I tap a link in an episode's description, such as a YouTube link, it is opened in Anytime. This is true for other links as well. I would like some ability to turn this behavior off. I initially thought websites and other apps I had were broken until I realized what was going on. It breaks expected behavior.

I also cannot copy links from an episode's description. This exacerbates the previously mentioned link-handling behavior. Not only can I not open links in other apps, but I also cannot copy links to manually paste them later. This means that I have to manually type links, switching between Anytime and wherever else I want to paste the link in.

**Comment by amugofjava** [comment]
Hi @Solomon1732,

Thanks for brining this issue to my attention. This is a bug as the correct behaviour should be that the link is opened in an external application rather than inside a view within Anytime. To help with testing, do you have an example Podcast that contains links to YouTube videos within an episode description or notes? Thanks.

**Reply by Solomon1732** [reply]
An example is the Saad Truth with Dr. Saad: https://podcastindex.org/podcast/1252140 . Most of his episodes have an external link. Episode 435 (from August 12, 2022) has one.
